bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -190,13 +190,11 @@ async fn main() -> Result<()> {
 fn load_configs() -> (AwsConfig, Arc<Config>) {
     // First load the configuration
     let aws_config = AwsConfig {
-        region: env::var("AWS_DEFAULT_REGION").expect("AWS_DEFAULT_REGION not set"),
-        aws_access_key_id: env::var("AWS_ACCESS_KEY_ID").expect("AWS_ACCESS_KEY_ID not set"),
-        aws_secret_access_key: env::var("AWS_SECRET_ACCESS_KEY")
-            .expect("AWS_SECRET_ACCESS_KEY not set"),
-        aws_session_token: env::var("AWS_SESSION_TOKEN").expect("AWS_SESSION_TOKEN not set"),
-        function_name: env::var("AWS_LAMBDA_FUNCTION_NAME")
-            .expect("AWS_LAMBDA_FUNCTION_NAME not set"),
+        region: env::var("AWS_DEFAULT_REGION").unwrap_or("us-east-1".to_string()),
+        aws_access_key_id: env::var("AWS_ACCESS_KEY_ID").unwrap_or("".to_string()),
+        aws_secret_access_key: env::var("AWS_SECRET_ACCESS_KEY").unwrap_or("".to_string()),
+        aws_session_token: env::var("AWS_SESSION_TOKEN").unwrap_or("".to_string()),
+        function_name: env::var("AWS_LAMBDA_FUNCTION_NAME").unwrap_or("".to_string()),
```

Comment:
I think an `unwrap_or_default` might be better here

Reply:
yup

Reply:
sorry, new baby brain. literally forgot about that method